lib/discover.js

```diff
@@ -48,7 +48,7 @@ TesselSeeker.prototype.start = function(opts) {
       .then(() => {
         debug('Connection opened:', tessel.connection.host);
         if (conn.connectionType === 'LAN' && !conn.authorized) {
-          tessel.name = conn.host.slice(0, -6);
+          tessel.name = conn.host.slice(0, -7);
```

Comment:
Above this line, write an explanation comment that includes an example of the full `conn.host` string, as well as the sliced resulting string.